### DISCUSSION TITLE
Async 1.9.1 Build

### DISCUSSION THREAD

**Discussion by ghost** [Sporif/dxvk-async#21] [General]
@Sporif 

Hi good sir. I was wondering if we could get the Async 1.9.1 build released as dxvk has been updated to 1.9.1 with lots of issues fixed

Thank you 🤠🤠🤠

**Comment by Sporif** [comment]
Just released it: https://github.com/Sporif/dxvk-async/releases/tag/1.9.1

**Reply by thinkprocess** [reply]
Thank you very much! You are the best!

**Reply by massatt212** [reply]
i read online u can get ban for Async or its safe?

**Reply by Sporif** [reply]
I've been using it for years in various multiplayer games without getting banned. I've never heard of anyone getting banned because they used dxvk-async.

**Reply by massatt212** [reply]
alright, i think ill adopt it
